### PR TITLE
[Sprint maintain] Check internet connection

### PR DESCRIPTION
**Pull request by tranminhtri911**
- https://dev.framgia.com/redmine/issues/137368
- Check internet connection
- Effect : None

### CODE REVIEW

File: app/src/main/java/com/framgia/wsm/utils/InternetManager.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+public class InternetManager {
+    private Context mContext;
+
+    public InternetManager(Context context) {
+        mContext = context;
+    }
+
+    public boolean isConnection() {
+        ConnectivityManager cm = (ConnectivityManager) mContext.
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/utils/InternetManager.java]: ERROR: '.' should be on a new line.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -381,4 +381,6 @@
     <string name="can_not_find_request">Can\'t find request!</string>
     <string name="approve">Approve</string>
     <string name="number_hour">Number hours</string>
+    <string name="no_network_connection">No Network Connection!</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`no_network_connection`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -381,4 +381,6 @@
     <string name="can_not_find_request">Can\'t find request!</string>
     <string name="approve">Approve</string>
     <string name="number_hour">Number hours</string>
+    <string name="no_network_connection">No Network Connection !</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`no_network_connection`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

**Comment by daolq3012** [comment]
LGTM

**Pull request merged**